hw/bsp/bbc_microbit/pkg.yml

```diff
@@ -28,54 +28,49 @@ pkg.keywords:
 
 pkg.cflags:
     # Nordic SDK files require these defines.
-    - '-DADC_ENABLED=1'
-    - '-DCLOCK_ENABLED=1'
-    - '-DCOMP_ENABLED=0'
-    - '-DEGU_ENABLED=0'
-    - '-DGPIOTE_ENABLED=1'
-    - '-DI2S_ENABLED=0'
-    - '-DLPCOMP_ENABLED=1'
     - '-DNRF51'
-    - '-DPDM_ENABLED=0'
-    - '-DPERIPHERAL_RESOURCE_SHARING_ENABLED=1'
-    - '-DPWM0_ENABLED=1'
-    - '-DPWM1_ENABLED=0'
-    - '-DPWM2_ENABLED=0'
-    - '-DQDEC_ENABLED=1'
-    - '-DRNG_ENABLED=1'
-    - '-DRTC0_ENABLED=0'
-    - '-DRTC1_ENABLED=0'
-    - '-DRTC2_ENABLED=0'
-    - '-DSAADC_ENABLED=0'
-    - '-DSPI0_CONFIG_MISO_PIN=28'
-    - '-DSPI0_CONFIG_MOSI_PIN=25'
-    - '-DSPI0_CONFIG_SCK_PIN=29'
-    - '-DSPI0_ENABLED=1'
-    - '-DSPI0_USE_EASY_DMA=0'
-    - '-DSPI1_ENABLED=0'
-    - '-DSPI2_ENABLED=0'
-    - '-DSPIS0_CONFIG_MISO_PIN=4'
-    - '-DSPIS0_CONFIG_MOSI_PIN=3'
-    - '-DSPIS0_CONFIG_SCK_PIN=2'
-    - '-DSPIS0_ENABLED=0'
-    - '-DSPIS1_CONFIG_MISO_PIN=28'
-    - '-DSPIS1_CONFIG_MOSI_PIN=25'
-    - '-DSPIS1_CONFIG_SCK_PIN=29'
-    - '-DSPIS1_ENABLED=1'
-    - '-DSPIS2_ENABLED=0'
-    - '-DTIMER0_ENABLED=1'
-    - '-DTIMER1_ENABLED=0'
-    - '-DTIMER2_ENABLED=0'
-    - '-DTIMER3_ENABLED=0'
-    - '-DTIMER4_ENABLED=0'
-    - '-DTWI0_CONFIG_SCL=0'
-    - '-DTWI0_CONFIG_SDA=1'
-    - '-DTWI0_ENABLED=1'
-    - '-DTWI1_ENABLED=0'
-    - '-DTWIS0_ENABLED=0'
-    - '-DTWIS1_ENABLED=0'
-    - '-DUART0_ENABLED=1'
-    - '-DWDT_ENABLED=1'
+    - '-DNRFX_CLOCK_ENABLED=0'
```

Comment:
These are stand in for the nordic driver template (in this case nrx_config.h) Note that were turning all of these 'off' as Im frankly not sure why any of them were enabled previously. They should only be enabled if were using nordic driver files, which ideally we shouldnt be except in cases of the ADC where we are wrapping their driver with ours.